drivers/hmis_external_apis/spec/models/hmis_external_apis/link_spec.rb

```diff
@@ -5,38 +5,103 @@
 ###
 
 require 'rails_helper'
-require 'webmock/rspec'
 
 # CAUTION: This is not part of the normal test suite. It runs against a live remote endpoint
 # We need many secrets to test this. Essentially, this runs locally or on staging
 RSpec.describe 'LINK API', type: :model do
   if ENV['OAUTH_CREDENTIAL_TEST'] == 'true'
-    let(:host) { 'example.com' }
+    let(:base_url) { ENV.fetch('LINK_BASE_URL') + '/' }
     let(:client_id) { ENV.fetch('LINK_CLIENT_ID') }
     let(:client_secret) { ENV.fetch('LINK_CLIENT_SECRET') }
     let(:token_url) { ENV.fetch('LINK_TOKEN_URL') }
     let(:oauth_scope) { 'GREEN_RIVER' }
     let(:ocp_apim_subscription_key) { ENV.fetch('LINK_OCP_APIM_SUBSCRIPTION_KEY') }
 
+    let(:requested_by) { 'user1@example.com' }
+    let(:now) { Time.now }
+    let(:program_id) { 867 }
+    let(:unit_type_id) { 20 }
+
+    def format_date(date)
+      date.to_s(:iso8601)
+    end
+
     let(:subject) do
       HmisExternalApis::OauthClientConnection.new(
         client_id: client_id,
         client_secret: client_secret,
         token_url: token_url,
         headers: { 'Ocp-Apim-Subscription-Key' => ocp_apim_subscription_key },
-        base_url: "https://#{host}/",
+        base_url: base_url,
         scope: oauth_scope,
       )
     end
 
-    it 'supports a get' do
-      # FIXME: the LINK API endpoints aren't built yet. This just tests auth
-      path = 'test/resources/1'
-      stub_request(:get, "#{subject.base_url}#{path}")
-        .to_return(status: 200, body: nil, headers: { 'Content-Type' => 'application/json' })
+    it 'creates referral request' do
+      payload = {
+        'requestedDate' => format_date(now),
+        'programID' => program_id,
+        'unitTypeID' => unit_type_id,
+        'estimatedDate' => format_date(now + 1.day),
+        'requestorName' => 'greenriver',
+        'requestorPhoneNumber' => '8028675309',
+        'requestorEmail' => 'test@greenriver.com',
+        'requestedBy' => requested_by,
+      }
+
+      result = subject.post('Referral/ReferralRequest', payload)
+      # {"message"=>"A server error occurred.", "detail"=>"ORA-00942: table or view does not exist", "errorReferenceId"=>"58e06257-3128-43d0-96fe-03cc6af4d8d6"}
```

Comment:
Fyi @ttoomey this one and voiding a referral request are working now. The other two both still have outstanding issues that have been communicated:

```
Unit/Capacity
Request: "{\"programID\":1008,\"unitTypeID\":20,\"availableUnits\":3,\"requestedBy\":\"[test@greenriver.com](mailto:test@greenriver.com)\"}"
Response: "{\"message\":\"No Resource found with the values provided. \",\"detail\":\"Invalid Parameters\"}"

Referral/PostingStatus
Request: "{\"postingId\":3777,\"postingStatusId\":18,\"deniedReasonId\":1,\"referralResultId\":158,\"statusNote\":\"test\",\"contactDate\":\"2023-05-02T17:41:49+00:00\",\"requestedBy\":\"[test@greenriver.com](mailto:test@greenriver.com)\"}"
Response: "{\"message\":\"Posting Status Id is not valid. \",\"detail\":\"Invalid Parameters.\"}"
```